### PR TITLE
Feat assignment 7 git in visual studio

### PR DESCRIPTION
**Pull request by aristovinceselvanathan**

### CODE REVIEW

File: src/Assignment7/Git/Readme.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Building a Simple Project with Git in Visual Studio
+## Objective: 
+The goal of this assignment is to familiarize yourself with the Git functionalities within Visual
+Studio. You will create a simple project, make changes, and use Git for version control.
+### Part I: Setting Up Your Environment
+1. Install Git and Visual Studio, if you haven't already.
+2. Open Visual Studio and createa new project. Choose a simple console application for this assignment.<br><br>
+![Alt text](<Screenshot (13).png>)
```
Comment (sandeep-sol) [inline src/Assignment7/Git/Readme.md]: Please update the name of the screenshots such that it describes what is shown.

File: src/Assignment7/Git/Screenshot (6)-1.png
Comment (sandeep-sol) [inline src/Assignment7/Git/Screenshot (6)-1.png:1]: This image is a duplicate. Is this intentional?

File: src/Assignment7/Git/Screenshot (10).png
Comment (sandeep-sol) [inline src/Assignment7/Git/Screenshot (10).png:1]: The task says that you should make several changes in a file and commit those changes selectively. But this has changes in 2 files and one of them is staged.

This is not expected. Please update it.

File: src/Assignment7/Git/Readme.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Overview
+The assignemnt that can able to understand the concept of the git in Visual Studio by the use of the git chages window. Became familar with the window.
+
+## Requirements
+1. Git Repository
+2. Visual Studio
+3. Project Files (Git Files)
+
+## Description about the methods in Program Class:
+Main method just prints the required informations like Hello World, How are you etc.
```
Comment (sandeep-sol) [inline src/Assignment7/Git/Readme.md]:
```suggestion
The main method just prints the required information like Hello World, How are you, etc.
```

File: src/Assignment7/Git/Readme.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## Implementation Considerations
+- Resolving the merge confilct between the branches
```
Comment (sandeep-sol) [inline src/Assignment7/Git/Readme.md]:
```suggestion
- Resolving the merge conflicts between the branches
```

File: src/Assignment7/Git/Readme.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- Staging the required files
+- Creating the new branch from the current branch
```
Comment (sandeep-sol) [inline src/Assignment7/Git/Readme.md]:
```suggestion
- Creating a new branch from the current branch
```

File: src/Assignment7/Git/Readme.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
+# Building a Simple Project with Git in Visual Studio
+## Objective: 
+The goal of this assignment is to familiarize yourself with the Git functionalities within Visual
+Studio. You will create a simple project, make changes, and use Git for version control.
+### Part I: Setting Up Your Environment
+1. Install Git and Visual Studio, if you haven't already.
+2. Open Visual Studio and createa new project. Choose a simple console application for this assignment.<br><br>
```
Comment (sandeep-sol) [inline src/Assignment7/Git/Readme.md]:
```suggestion
2. Open Visual Studio and create a new project. Choose a simple console application for this assignment.<br><br>
```

File: src/Assignment7/Git/Readme.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+### Part 2: Initializing a Git Repository
+1. Initialize a new Git repository for your project in Visual Studio.
+2. Make an initial commit with the project's starting code.<br><br>
+![Alt text](<Screenshot (2).png>)
+### Part 3: Making and Staging Changes
+1. Make some changes to your code. This could be adding a new feature or modifying an existing one.
+2. Stage these changes using the 'Changes' window in Visual Studio.<br><br>
+![Alt text](<Screenshot (5).png>)
+### Part 4: Committing Changes
+1. Writea meaningful commit message describing the changes you made.
```
Comment (sandeep-sol) [inline src/Assignment7/Git/Readme.md]:
```suggestion
1. Write a meaningful commit message describing the changes you made.
```

File: src/Assignment7/Git/Readme.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+2. Identify the changes made in each commit.<br><br>
+![Alt text](<Screenshot (8).png>)
+## Part 8: Selective Staging
+1. Make several changes to a file in your project.
+2. Use selective staging to stage only some of these changes and commit them.
+3. Stage and commit the remaining changes separately.<br><br>
+![Alt text](<Screenshot (10).png>)<br>
+# Documentation Report
+## Overview
+The assignemnt that can able to understand the concept of the git in Visual Studio by the use of the git chages window. Became familar with the window.
```
Comment (sandeep-sol) [inline src/Assignment7/Git/Readme.md]: Please update this statement.

File: src/Assignment7/Git/Readme.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+![Alt text](<Screenshot (17).png>)<br>
+### Part 7: Viewing Git History
+1. View the commit history of your repository.
+2. Identify the changes made in each commit.<br><br>
+![Alt text](<Screenshot (8).png>)
+## Part 8: Selective Staging
+1. Make several changes to a file in your project.
+2. Use selective staging to stage only some of these changes and commit them.
+3. Stage and commit the remaining changes separately.<br><br>
+![Alt text](<Screenshot (10).png>)<br>
```
Comment (sandeep-sol) [inline src/Assignment7/Git/Readme.md]: It's good that you've added all these screenshots for reference.

File: src/Assignment7/Git/Screenshot (10).png
Comment (sandeep-sol) [inline src/Assignment7/Git/Screenshot (10).png:1]: Move all the screenshots to a folder.

**Review by sandeep-sol — CHANGES_REQUESTED** [review]

File: src/Assignment7/Git/Readme.md (hunk 1 above)
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Readme.md]: I have resolved it

File: src/Assignment7/Git/Screenshot (6)-1.png
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Screenshot (6)-1.png:1]: I have removed the file

File: src/Assignment7/Git/Screenshot (10).png
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Screenshot (10).png:1]: I have changed the screenshot

File: src/Assignment7/Git/Readme.md (hunk 2 above)
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Readme.md]: I have changed the statement

File: src/Assignment7/Git/Readme.md (hunk 3 above)
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Readme.md]: I have changed the statement

File: src/Assignment7/Git/Readme.md (hunk 4 above)
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Readme.md]: I have changed the statement

File: src/Assignment7/Git/Readme.md (hunk 5 above)
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Readme.md]: I have changed the statement

File: src/Assignment7/Git/Readme.md (hunk 6 above)
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Readme.md]: I Have changed the statement

File: src/Assignment7/Git/Readme.md (hunk 7 above)
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Readme.md]: I have updated the statement

File: src/Assignment7/Git/Screenshot (10).png
Comment (aristovinceselvanathan) [reply src/Assignment7/Git/Screenshot (10).png:1]: I have moved the folder

**Review by sandeep-sol — APPROVED** [review]

**Pull request merged**